packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp

```diff
@@ -684,40 +744,77 @@ void LayoutAnimationsProxy::startExitingAnimation(
 
 void LayoutAnimationsProxy::startLayoutAnimation(
```

Comment:
How about introducing a new method called `startLayoutTransition`? 🤔

Reply:
Do you mean `startStyleTransition` or do you think of renaming all the stuff related to layout Transitions to use naming `LayoutTransition` instead of `LayoutAnimation`?